package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Do we need this file?

Reply:
Past PRs only have mixer diff - https://github.com/datacommonsorg/website/pull/1681

Reply:
Yup, removed!